frontend/src/__tests__/cypress/cypress/e2e/projects/ProjectList.cy.ts

```diff
@@ -0,0 +1,161 @@
+import { mockDashboardConfig } from '~/__mocks__/mockDashboardConfig';
+import { mockStatus } from '~/__mocks__/mockStatus';
+import {
+  mockProjectK8sResource,
+  mockProjectK8sDeletedResource,
+} from '~/__mocks__/mockProjectK8sResource';
+import { mockK8sResourceList } from '~/__mocks__/mockK8sResourceList';
+import { createProjectModal, projectListPage } from '~/__tests__/cypress/cypress/pages/projects';
+import { deleteModal } from '~/__tests__/cypress/cypress/pages/components/DeleteModal';
+
+describe('Data science projects details', () => {
+  it('should start with an empty project list', () => {
+    initIntercepts();
+    initEmptyProjectIntercepts();
+    cy.visitWithLogin('/projects');
+    projectListPage.shouldBeEmpty();
+  });
+
+  it('should open a modal to create a project', () => {
+    initIntercepts();
+    initEmptyProjectIntercepts();
+    cy.visitWithLogin('/projects');
+    projectListPage.findCreateProjectButton().click();
+    createProjectModal.shouldBeOpen();
+    createProjectModal.findCancelButton().click();
+    createProjectModal.shouldBeOpen(false);
+    projectListPage.findCreateProjectButton().click();
+    createProjectModal.shouldBeOpen();
+    createProjectModal.findSubmitButton().should('be.disabled');
+  });
+
+  it('should create project', () => {
+    initIntercepts();
+    initCreateProjectIntercepts();
+
+    projectListPage.visit();
+    projectListPage.navigate();
+    projectListPage.findCreateProjectButton().click();
+    createProjectModal.findNameInput().type('My Test Project');
+    createProjectModal.findDescriptionInput().type('Test project description.');
+    createProjectModal.findSubmitButton().should('be.enabled');
+    createProjectModal.findResourceNameInput().should('have.value', 'my-test-project').clear();
+    createProjectModal.findResourceNameInput().should('have.attr', 'aria-invalid', 'true');
+    createProjectModal.findSubmitButton().should('be.disabled');
+    createProjectModal.findResourceNameInput().type('test-project');
+
+    cy.intercept(
+      '/api/k8s/apis/project.openshift.io/v1/projects',
+      mockK8sResourceList([mockProjectK8sResource({})]),
+    ).as('createdProjects');
+
+    createProjectModal.findSubmitButton().click();
+    cy.wait('@createdProjects');
```

Comment:
It's important to assert the payload of non `GET` requests to ensure we're sending the correct data.

Reply:
Sure.I will add payload assertions